server/channels/api4/job_test.go

```diff
@@ -36,7 +36,11 @@ func TestCreateJob(t *testing.T) {
 	t.Run("valid job as user with permissions", func(t *testing.T) {
 		received, _, err := th.SystemAdminClient.CreateJob(context.Background(), job)
 		require.NoError(t, err)
-		defer th.App.Srv().Store().Job().Delete(received.Id)
+		defer func() {
+			if result, appErr := th.App.Srv().Store().Job().Delete(received.Id); appErr != nil {
+				t.Logf("Failed to delete job (result: %v): %v", result, appErr)
+			}
```

Comment:
Let's use `require.Nil` to be consistent with the rest of the code base.

Reply:
Yes, should be fixed